src/admin-views/aggregator/status.php

```diff
@@ -90,22 +100,47 @@
 
 		if ( ! $up || is_wp_error( $up ) ) {
 			$indicator = 'bad';
-			$notes = __( 'The Event Aggregator server is not currently responding', 'the-events-calendar' );
+			$text = __( 'Not connected to', 'the-events-calendar' ) . ' ' . $ea_server;
```

Comment:
The word order may not make sense when translated. Can we use a `sprintf()` and use a string placeholder for `$ea_server`?